governance/blog-guidelines.md

```diff
@@ -0,0 +1,108 @@
+# Blog Guidelines
+
+## Introduction
+
+Welcome to the Security TAG (STAG) Blog! Our blog is a platform for sharing insights, experiences, and knowledge about Cloud Native software and the community. We encourage contributions from community members to help educate, inspire, and engage others. To ensure quality and consistency, we have established the following guidelines for content creation, submission review, and the overall value to the community.
+
+## Content Creation
+
+### Topics
+
+- **Cloud Native Projects**: Share updates, features, and success stories.
+- **Technical Tutorials**: Provide how-to guides, coding tips, and best practices.
+- **Community Stories**: Highlight contributors, events, and community achievements.
+- **Opinion Pieces**: Discuss trends, challenges, and the future of open source.
+
+### Writing Style
+
+- **Clarity**: Write in clear, concise language. Avoid jargon unless it's well-explained.
+- **Engagement**: Make your post interesting and engaging. Use anecdotes and examples.
+- **Accuracy**: Ensure all information is accurate and up-to-date.
+- **Inclusivity**: Use inclusive language and be respectful to all readers.
+
+### Formatting
+
+- **File Name**: Blog Files should be dash-case `.md` markdown filetype.
+- **Headings**: Use appropriate headings (H1, H2, H3, etc.) to structure your post.
+- **Paragraphs**: Keep paragraphs short and focused.
+- **Lists**: Use bullet points or numbered lists to highlight key points.
+- **Tags**: Include relevant tags in the blog for filtering capabilities.
+- **Code**: Use code blocks for any code snippets.
+- **Images**: Include relevant images and graphics to enhance the content.
+
+### Front Matter
+
+The start of each blog document should contain the `frontmatter` information.
+
+Required `frontmatter` fields:
+
+- **title**: Title-case title for the blog post
+- **author**: Name of blog post author
+
+Note: **date** can be included but will be appended to the `frontmatter` when publication date is decided.
+
+#### Example
+
+```yaml
+---
+title:  "New Informative Blog Post"
+date:   2024-06-26 09:00:00 -0700
+author: R. Raccoon
+---
+```
+
+### Length
+
+- Aim for 500-1200 words. Ensure the content is comprehensive yet concise.
+
+## Submission Review Process
+
+### Submission
+
+1. **Draft**: Write your blog post in a markdown file under `tag-security/website/content/blog`
+2. **Submission Pull Request**: Submit your draft using the [blog submission template](https://github.com/cncf/tag-security/blob/main/.github/ISSUE_TEMPLATE/blog-submission.md).
```

Comment:
```suggestion
2. **Submission Pull Request**: Submit your draft using the [blog submission template](/.github/ISSUE_TEMPLATE/blog-submission.md).
```

This should fix the broken link check. Though I'm not sure whether or not it needs the leading `/`

Reply:
This is true - although it might haunt us on the website

Reply:
Looks like it worked!

Reply:
Open to making this change - need to think more about links that point to locations not included in the website

Reply:
Heh. I need to stop reviewing PRs in middle of long travel legs— thought you already applied the change here. If it doesn't work, we can just merge it with the hard path.